app/database/queries/breeze_buddy/template.py

```
@@ -149,6 +149,36 @@ def get_template_by_id_query(template_id: str) -> Tuple[str, List[Any]]:
+        # Filter by inbound_enabled in configurations JSON
+        # COALESCE ensures missing key defaults to FALSE
+        conditions.append(
+            "COALESCE((configurations->>'inbound_enabled')::boolean, FALSE) = TRUE"
+        )
+
+    query = f"""
+        SELECT id, merchant_id, shop_identifier, name, flow, expected_payload_schema, expected_callback_response_schema, configurations, outbound_number_id, is_active, created_at, updated_at
+        FROM {TEMPLATE_TABLE}
+        WHERE {' AND '.join(conditions)}
+        LIMIT 1
+    """
```

Ruff (0.14.14)

Possible SQL injection vector through string-based query construction

(S608)